projects/packages/search/changelog/update-search-move-record-meter

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I _think_ we're better off with `Significance: patch` as this doesn't change anything for the end user (feature isn't released yet). If we use `minor` we bump the version of the package and it's a whole thing.

cc @Automattic/jetpack-search to check my assumption